CADETPythonSimulator/unit_operation.py

```diff
@@ -928,64 +930,71 @@ def compute_residual(
 
         # Number of cake
 
-        self.residuals['cake']['n_in'] = n_cake_dot - rejection * n_feed_dot
+        self.residuals['cake']['n_in'] = n_cake_in_dot - rejection * n_feed_dot
 
         # Number of Permeate
 
         self.residuals['permeate_tank']['n_in'] =\
-            n_permeate_dot - (1 - rejection) * n_feed_dot
```

Comment:
Sorry if I was not clear, but I think we should *always* use an indentation for multi-line comments.

```
self.residuals['permeate_tank']['n_in'] = \
    n_permeate_dot - (1 - rejection) * n_feed_dot
```
Also, note the additional space before the backslash `\`

Alternatively, you can also use parenthesis:
```
self.residuals['permeate_tank']['n_in'] = (
    n_permeate_dot - (1 - rejection) * n_feed_dot
)
```